### DISCUSSION TITLE
allow control of overflow styling

### DISCUSSION THREAD

**Discussion by ryanchesler** [h2oai/wave#727] [Q&A]
Having an issue with a navbar that keeps getting assigned overflow scrolling for some reason. Can't seem to make it go away. In the top right the navbar gets assigned a scrollbar even though it doesn't seem necessary. Not sure if I am doing something wrong with my layout, but would like some way to correct it. Either by disabling overflow like I show in the second picture or just automatically via some layout correction. 

images of scrollbar and its removal via turning off overlow in css
<img width="1919" alt="Screen Shot 2021-04-09 at 12 58 04 PM" src="https://user-images.githubusercontent.com/26419230/114234356-7f1e3500-9933-11eb-92ba-73f77146f2b8.png">
<img width="1915" alt="Screen Shot 2021-04-09 at 12 57 54 PM" src="https://user-images.githubusercontent.com/26419230/114234365-834a5280-9933-11eb-96d4-7ef561fe5ecf.png">

layout code: https://github.com/h2oai/wave-document-ai/blob/v0.1/doc_ai/layouts.py

**Comment by mturoci** [comment]
Hi @ryanchesler, the link to layout code is dead (or maybe the repo is just private), but the solution is to slightly increase `zone` height.
